src/components/Content/Content.js

```diff
@@ -1,15 +1,214 @@
+          {formatSection(parametersB, activeSection, activeSubsection, setActiveSubsection, variables, setVariables)}
+        </React.Fragment>
+      );
+    } else {
+      return formatSection(section, activeSection, activeSubsection, setActiveSubsection, variables, setVariables);
+    }
+  });
+};
+
+const formatSection = (section, activeSection, activeSubsection, setActiveSubsection, variables, setVariables) => {
+  const key = section[0];
+  const value = section[1];
+
+  let labelKey = "label";
+  let contentKey = "content";
+  if (key === "parameters-b") {
+    labelKey = "labelB";
+    contentKey = "contentB";
+  }
+  return (
+    <FormGroup key={key} className={`fieldset ${activeSection === key ? "active" : ""}`}>
+      <h2>{value[labelKey]}</h2>
+      <p>{value[contentKey]}</p>
+
+      <div className="subNavigation">
+        {Object.entries(value.fields).map(field => {
+          const key = field[0];
+
+          return (
+            <button 
+              key={key}
+              onClick={() => setActiveSubsection(key)}
+              className={key === activeSubsection ? "active" : "" } 
+            >
+              {key}
+            </button>
+          );
+        })}
+      </div>
+      <div className="subNavigationContent">
+        {Object.entries(value.fields).map(field => {
+          const key = field[0];
+          const value = field[1];
+
+          if (activeSubsection !== key) {
+            return null;
+          }
+
+          return (
+            <section key={key} className="Questions">
+              {Object.entries(value).map(question => {
+                return formatQuestion(question, variables, setVariables);
+              })}
+            </section>
+            );
+          })
+        }
+      </div>
+    </FormGroup>
+  );
+}
+
+const formatNavigation = (data, activeSection, setActiveSection) => {
+  if (data == null) {
+    return;
+  }
+  return Object.entries(data).map(section => {
+    let key = section[0];
+
+    // If Parameters, export both A and B
+    if (key === "parameters") {
+      return (
+        <React.Fragment key="parameters">
+          <button 
+            className={`${section[0]}-a` === activeSection ? "active" : "" } 
+            key={section[0]+"a"} 
+            onClick={() => setActiveSection(section[0]+"-a")}
+          >
+            Parameters A
+          </button>
+          <br />
+          <button 
+            className={`${section[0]}-b` === activeSection ? "active" : "" } 
+            key={section[0]+"b"} 
+            onClick={() => setActiveSection(section[0]+"-b")}
+          >
+            Parameters B
+          </button>
+          <br />
+        </React.Fragment>
+      )
+    } else {
+      return (
+        <React.Fragment key={section[0]}>
+          <button
+            className={section[0] === activeSection ? "active" : "" } 
+            onClick={() => setActiveSection(section[0])}
+          >
+            {section[0]}
```

Comment:
Should use set key variable
```suggestion
            className={`${key}-a` === activeSection ? "active" : "" } 
            key={key+"a"} 
            onClick={() => setActiveSection(key+"-a")}
          >
            Parameters A
          </button>
          <br />
          <button 
            className={`${key}-b` === activeSection ? "active" : "" } 
            key={key+"b"} 
            onClick={() => setActiveSection(key+"-b")}
          >
            Parameters B
          </button>
          <br />
        </React.Fragment>
      )
    } else {
      return (
        <React.Fragment key={key}>
          <button
            className={key === activeSection ? "active" : "" } 
            onClick={() => setActiveSection(key)}
          >
            {key}
```